assignments/prototypes.js

```diff
@@ -7,8 +7,42 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
+  const GameObject = function(attributes){
+    this.createdAt = 'June52019',
```

Comment:
There is your date issue. We want to set it from the attributes, like the other properties. Then we can pass in a date object.